doc/source/release/1.18.0-notes.rst

```diff
@@ -1,8 +1,348 @@
-The NumPy 1.18 release is currently in development. Please check
-the ``numpy/doc/release/upcoming_changes/`` folder for upcoming
-release notes.
-The ``numpy/doc/release/upcoming_changes/README.txt`` details how
-to add new release notes.
-
-For the work in progress release notes for the current development
-version, see the `devdocs <https://numpy.org/devdocs/release.html>`__.
+================================
+NumPy NumPy 1.18.0 Release Notes
+================================
+
+
+New functions
+=============
+
+Multivariate hypergeometric distribution added to `numpy.random`
+----------------------------------------------------------------
+The method `multivariate_hypergeometric` has been added to the class
+`numpy.random.Generator`.  This method generates random variates from
+the multivariate hypergeometric probability distribution.
+(`gh-13794 <https://github.com/numpy/numpy/pull/13794>`__)
+
+
+Deprecations
```

Comment:
The order of headings is odd and inconsistent with previous release notes. Can this be ordering like we had it before?

Reply:
It is the same ordering when I check against the `doc/source/release/template.rst`, but that hasn't been revisited for some time.